### DISCUSSION TITLE
Automatically account verification after following verify_account_email_link

### DISCUSSION THREAD

**Discussion by bshourse** [janko/rodauth-rails#283] [Q&A] [answered]
Hello, Janko!

May you please give advice, is it possible to automatically verify account, after user clicks the verification link from email?
Right now, when user clicks link in email(`@rodauth.verify_account_email_link`), verify_account view renders and user should click `rodauth.verify_account_button`.

Maybe i can write some code in `verify_account_route` implement such logic there. And then use `verify_account` method from Rodauth module?

My expected result:
If account exists, not verified yet, and keys matched then i can explicitly verify account, and make redirect where i want(using `verify_account_redirect`), otherwise throw some error.

**Comment by janko** [comment, marked as answer]
Yes, you can automatically submit the form with JavaScript when the verification page is loaded, see https://github.com/janko/rodauth-rails/wiki/Autosubmit-Forms.

**Reply by bshourse** [reply]
Thank you for the idea!
